app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="devise_new_session">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
```

Comment:
[REQUIRED] Kindly remove the remember me checkbox since this is not present in the wireframe
![image](https://github.com/jugosack/blog-app-rails/assets/62938735/5dabe58a-3692-4b3d-b929-fec6added255)